docs/tutorials/deploying-an-application.md

```diff
@@ -10,136 +10,111 @@ This tutorial will take you through the process of deploying an App to a product
-    2020-01-01T00:00:00Z system/k8s/atom/app Status: Updating => Running
+    Sending build context to Docker daemon  48.96MB
+    Step 1/5 : FROM node:10.16.3-alpine
+    10.16.3-alpine: Pulling from library/node
+    e7c96db7181b: Pulling fs layer
+    50958466d97a: Pulling fs layer
+    56174ae7ed1d: Pulling fs layer
+    284842a36c0d: Pulling fs layer
+    284842a36c0d: Waiting
+    e7c96db7181b: Verifying Checksum
+    e7c96db7181b: Download complete
+    56174ae7ed1d: Verifying Checksum
+    56174ae7ed1d: Download complete
+    50958466d97a: Verifying Checksum
+    50958466d97a: Download complete
+    e7c96db7181b: Pull complete
+    284842a36c0d: Verifying Checksum
+    284842a36c0d: Download complete
+    50958466d97a: Pull complete
+    56174ae7ed1d: Pull complete
+    284842a36c0d: Pull complete
+    Digest: sha256:77c898d0da5e7bfb6e05c9a64de136ba4e03889a72f3c298e95df822a38f450d
+    Status: Downloaded newer image for node:10.16.3-alpine
+    ---> b95baba1cfdb
+    Step 2/5 : WORKDIR /usr/src/app
+    ---> Running in 19306c264a03
+    Removing intermediate container 19306c264a03
+    ---> 83aac0816456
+    Step 3/5 : COPY . /usr/src/app
+    ---> 6c88d15e88c9
+    Step 4/5 : EXPOSE 3000
+    ---> Running in b99318863aa8
+    Removing intermediate container b99318863aa8
+    ---> db735909e450
+    Step 5/5 : CMD ["node", "app.js"]
+    ---> Running in f1df464405be
+    Removing intermediate container f1df464405be
+    ---> 7bd8ca94d031
+    Successfully built 7bd8ca94d031
+    Successfully tagged 81719b4e5da8a821215126cc7edefe7fc2e5751f42db0d920ddbf574:latest
+    Running: docker tag 81719b4e5da8a821215126cc7edefe7fc2e5751f42db0d920ddbf574 do-test/nodejs:web.BNLGVRXMCAP
+    Running: docker tag do-test/nodejs:web.BNLGVRXMCAP registry.05456db021737ab6.convox.cloud/nodejs:web.BNLGVRXMCAP
+    Running: docker push registry.05456db021737ab6.convox.cloud/nodejs:web.BNLGVRXMCAP
+
+    Promoting RHQFJKRJNHJ... 
+    2020-05-14T21:37:13Z system/k8s/atom/app Status: Running => Pending
+    2020-05-14T21:37:14Z system/k8s/web-749dd486d8 Created pod: web-749dd486d8-8v4ss
+    2020-05-14T21:37:14Z system/k8s/web-749dd486d8-8v4ss Successfully assigned do-test-nodejs/web-749dd486d8-8v4ss to do-test-node-3n6m6
+    2020-05-14T21:37:14Z system/k8s/web Scaled up replica set web-749dd486d8 to 1
+    2020-05-14T21:37:15Z system/k8s/atom/app Status: Pending => Updating
+    2020-05-14T21:37:16Z system/k8s/web-749dd486d8-8v4ss Pulling image "registry.05456db021737ab6.convox.cloud/nodejs:web.BNLGVRXMCAP"
+    2020-05-14T21:37:22Z system/k8s/web-749dd486d8-8v4ss Successfully pulled image "registry.05456db021737ab6.convox.cloud/nodejs:web.BNLGVRXMCAP"
+    2020-05-14T21:37:22Z system/k8s/web-749dd486d8-8v4ss Created container main
+    2020-05-14T21:37:22Z system/k8s/web-749dd486d8-8v4ss Started container main
+    2020-05-14T21:37:27Z system/k8s/atom/app Status: Updating => Running
     OK
 
 
```

Comment:
The next paragraph should change `rails` to `nodejs`

Reply:
And if you're doing a `deploy` you don't need another terminal to run the services command as deploy will finish you and you get your prompt back!